.travis.yml

```diff
@@ -20,9 +20,36 @@ jobs:
         - cd backend
       install:
         - npm install
+        - cd node_modules/postman-templating
+        - echo $PWD
+        - npm install
+        - npm run test
       script:
+        - cd ../../
+        - echo $PWD
         - npm run lint-no-fix
+        - npm run build
         - npm run test
+      before_deploy:
+        - TAG=$REPO:travis-$TRAVIS_BRANCH-$TRAVIS_COMMIT-$TRAVIS_BUILD_NUMBER
```

Comment:
For consistency with workers, I think it would be better to put this into a separate shell script (`deploy.sh`).

Reply:
Hmmmm so i tried using shell script for before_deploy and it doesn't seem to work.

Reply:
We could just put everything in a shell script and run deploy, which I have not tried yet